src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ApprenticeshipIncentive.cs

```diff
@@ -24,6 +24,8 @@ public sealed class ApprenticeshipIncentive : AggregateRoot<Guid, Apprenticeship
         public PendingPayment NextDuePayment => GetNextDuePayment();
         public IReadOnlyCollection<Payment> Payments => Model.PaymentModels.Map().ToList().AsReadOnly();
 
+        public bool Withdrawn => Model.Withdrawn;
```

Comment:
Can keep this private if the above change is done.  This avoids coupling on this method.